src/utils/nviHelpers.ts

```diff
@@ -85,9 +85,10 @@ const hasChangedContributorsOrAffiliations = async (
 };
 
 export const willResetNviStatuses = async (persistedRegistration: Registration, updatedRegistration: Registration) => {
-  const canBeNviCandidate = nviApplicableTypes.includes(
-    persistedRegistration.entityDescription?.reference?.publicationInstance?.type ?? ''
-  );
+  const canBeNviCandidate =
+    nviApplicableTypes.findIndex(
+      (category) => category === persistedRegistration.entityDescription?.reference?.publicationInstance?.type
+    ) >= 0;
```

Comment:
Bare fint å få det på riktig type, så det er bra!
Kan bevare den gamle sjekken om vi bare sjekker at det er en verdi først også, da type-feilen kommer pga nullable type

```suggestion
  const canBeNviCandidate =
    !!persistedRegistration.entityDescription?.reference?.publicationInstance?.type &&
    nviApplicableTypes.includes(persistedRegistration.entityDescription?.reference?.publicationInstance?.type);
```